doc/ref_model/chapters/chapter07.md

```diff
@@ -82,43 +82,30 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor i
 
 With that in mind, the NFVI reference model and the supporting architectures are not only required to optimally support networking functions, but they must be designed with common security principles and standards from inception.  These best practices must be applied at all layers of the infrastructure stack and across all points of interconnections with outside networks, APIs and contact points with the NFV network functions overlaying or interacting with that infrastructure.
-Standards organizations with recommendations and best practices, and certifications that need to be taken into consideration include the following examples. However this is by no means an exhaustive list, just some of the more important standards in current use.
-
- • Center for Internet Security - https://www.cisecurity.org/
-
- • Cloud Security Alliance - https://cloudsecurityalliance.org/
-
- • Open Web Application Security Project https://www.owasp.org
-
- • The National Institute of Standards and Technology (NIST) (US Only)
-
- • FedRAMP Certification https://www.fedramp.gov/ (US Only)
-
- • ETSI Cyber Security Technical Committee (TC CYBER) - https://www.etsi.org/committee/cyber
-
- • ETSI Industry Specification Group Network Functions Virtualisation (ISG NFV) - https://www.etsi.org/committee/1427-nfv
-
-• ISO (the International Organization for Standardization) and IEC (the International Electrotechnical Commission) - www.iso.org.  The following ISO standards are of particular interest for NFVI
-
-    o ISO/IEC 27002:2013 - ISO/IEC 27001 is the international Standard for best-practice information security management systems (ISMSs).
-
-    o ISO/IEC 27032 - ISO/IEC 27032is the international Standard focusing explicitly on cybersecurity.
-
-    o ISO/IEC 27035 - ISO/IEC 27035 is the international Standard for incident management. Incident management
-
-    o ISO/IEC 27031 - ISO/IEC 27031 is the international Standard for ICT readiness for business continuity.
+Standards organizations with recommendations and best practices, and certifications that need to be taken into consideration include the following examples. However this is by no means an exhaustive list, just some of the more important standards in current use. [inde](http://index.hu)
```

Comment:
\[inde](http://index.hu) what is this?

Reply:
@deebhatia , :)  we can remove it, the clause itself not clear we can remove the clause

Reply:
Uh-oh, sorry I was just testing the syntax of linking ;)

Reply:
Corrected now.  Thanks for pointing it out.

Reply:
> Uh-oh, sorry I was just testing the syntax of linking ;)

No worries. 😄